UnitTestFrameworkPkg/Library/UnitTestPersistenceLibSimpleFileSystem/UnitTestPersistenceLibSimpleFileSystem.c

```diff
@@ -146,9 +148,10 @@ GetCacheFileDevicePath (
     FreePool (AppPath);
   }
 
-  if (CacheFilePath != NULL) {
-    FreePool (CacheFilePath);
-  }
+  // if (CacheFilePath != NULL) {
+  //   FreePool (CacheFilePath);
+  // }

```

Comment:
similarly, get rid of commented out section or add some MU_CHANGE tags if we want to track the deviation?

Reply:
Should be MU_CHANGE tags to be consistent with our process.

Reply:
Labeled with MU_CHANGE for the lines touched here.